charts/jaeger/templates/agent-ds.yaml

```diff
@@ -28,6 +28,12 @@ spec:
         {{- toYaml .Values.agent.podLabels | nindent 8 }}
 {{- end }}
     spec:
+      updateStrategy:
+        type: {{ .Values.agent.daemonset.updateStrategy }}
+        {{- if eq .Values.agent.daemonset.updateStrategy "RollingUpdate" -}}
+        rollingUpdate:
+          maxUnavailable: {{ .Values.agent.daemonset.maxUnavailable }}
+        {{- end }}
```

Comment:
I think an approach like https://github.com/fluent/helm-charts/blob/e6805d4509868b76717677c7f7eecc4834d49d5a/charts/fluent-bit/templates/daemonset.yaml#L12-L15 will be more flexible and removes unneeded abstraction, WDYT?

Reply:
Makes sense @naseemkullah 
Certainly makes it easier. 😄

Reply:
Made the change.